src/openapi.json

```diff
@@ -348,6 +348,44 @@
           "title",
           "createdAt"
         ]
+      },
+      "Bundle": {
+        "type": "object",
+        "properties": {
+          "entries": {
+            "type": "array",
+            "description": "The collection of requested resources."
+          },
+          "total": {
+            "type": "integer",
+            "description": "The total number of requested resources that exist in the database, across all pages."
+          }
```

Comment:
Right now, we are not providing any placeholder data for this field, which apparently inspires OpenAPI to just dream up `total: 0` in its "Example Value" section. It's super cool that it can do that! ...but perhaps somewhat unhelpful in this specific case. Would it be hard to add a non-zero example?

Reply:
Super easy -- I'll add that now.